packages/styles/src/layout/index.scss

```
@@ -1,2 +1,3 @@
+@forward './sections';
```

I'd call it singular "section.scss" instead of plural.

```suggestion
@forward './section';
```